src/pytest_mysql/factories/client.py

```diff
@@ -66,7 +66,7 @@ def _connect(connect_kwargs: dict, query_str: str) -> MySQLdb.Connection:
         except ProgrammingError as e:
             if "database exists" in str(e):
                 raise DatabaseExists(
-                    f"Database {mysql_db} already exists. There's some test "
+                    f"Database {query_str} already exists. There's some test "
```

Comment:
query_str would only make it less readable 🤔  We need to pass the correct database name inside to get a clear message. Can be solved separately from this pull request though.

Reply:
I've fixed this also